packages/material-components-web/material-components-web.scss

```diff
@@ -43,6 +43,7 @@
 @use "@material/notched-outline/mdc-notched-outline";
 @use "@material/radio/mdc-radio";
 @use "@material/ripple/mdc-ripple";
+@use "@material/segmented-button/styles";
```

Comment:
Seems like dependency check does not like `/styles` pattern. Sending a fix now.

Reply:
Sent PR #6152 to fix CSS dependency test.

You'll have to rebase your branch after above PR is merged.

Reply:
PR #6152 is now merged. Rebase your branch to fix the failing test.

Reply:
Thanks for the patch!